src/portable/portabled-image-bus.c

```diff
@@ -223,12 +225,14 @@ int bus_image_common_attach(
                 Image *image,
                 sd_bus_error *error) {
 
-        _cleanup_strv_free_ char **matches = NULL;
+        _cleanup_strv_free_ char **matches = NULL, **extension_images = NULL;
         PortableChange *changes = NULL;
         PortableFlags flags = 0;
         const char *profile, *copy_mode;
         size_t n_changes = 0;
         int runtime, r;
+        /* Unused for now, but added to the DBUS methods for future-proofing */
+        uint64_t input_flags = 0;
```

Comment:
I don't think this is necessary.
sd_bus_message_read(3) says:
> Each argument may also be NULL, in which case the value is read and ignored.

But I think that we don't need to read the value to verify message integrity. We can just ignore it, and just rely on the signature of the message being checked. @poettering?

Reply:
(that said, I think we should read the param, and refuse if non-null, so that later on clients can check if we support some flag sanely)

Reply:
changed to check and reject flags != 0